### ISSUE TITLE
ubuntu18.04 guacamole make时出问题

### ISSUE THREAD

**Issue by gitren** [jumpserver/jumpserver#1994]
- ubuntu server 18.04

## guacamole make时出问题

make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/opt/docker-guacamole/guacamole-server-0.9.14/src/common-ssh'
                Making all in src/terminal
make[2]: Entering directory '/opt/docker-guacamole/guacamole-server-0.9.14/src/terminal'
               CC       libguac_terminal_la-typescript.lo
typescript.c: In function ‘guac_terminal_typescript_alloc’:
typescript.c:133:46: error: ‘%s’ directive writing 6 bytes into a region of size between 0 and 2047 [-Werror=format-overflow=]
     sprintf(typescript->timing_filename, "%s.%s", typescript->data_filename,
                                                                            ^~
In file included from /usr/include/stdio.h:862:0,
                 from typescript.c:28:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:33:10: note: ‘__builtin___sprintf_chk’ output between 8 and 2055 bytes into a destination of size 2048
   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       __bos (__s), __fmt, __va_arg_pack ());
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Makefile:567: recipe for target 'libguac_terminal_la-typescript.lo' failed
make[2]: *** [libguac_terminal_la-typescript.lo] Error 1
make[2]: Leaving directory '/opt/docker-guacamole/guacamole-server-0.9.14/src/terminal'
Makefile:504: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/opt/docker-guacamole/guacamole-server-0.9.14'
Makefile:428: recipe for target 'all' failed
make: *** [all] Error 2


![image](https://user-images.githubusercontent.com/3930329/47976671-75eb6700-e0ee-11e8-9102-8d6c8cf6992c.png)

**Comment by wojiushixiaobai** [comment]
@gitren 请尝试用docker部署组件，我这周验证一下然后会给出相应的解决方案。麻烦老哥了

**Comment by gitren** [comment]
为什么closed




| |
L-sheng
邮箱：obwxsln@163.com
|

Signature is customized by Netease Mail Master

在2018年11月26日 14:44，老广 写道：

Closed #1994.

**Comment by wojiushixiaobai** [comment]
@gitren 上面已经给出解决方案，我们推荐使用centos7部署
